src/labels/CustomLabels.labels

```diff
@@ -1284,8 +1284,8 @@
         <categories>Setup</categories>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>Recommended fix when Health Check can&apos;t find a mapping&apos;s Auto-Enrollment Role</shortDescription>
```

Comment:
should we keep the &apos;? I dont quite remember which one we have been using

Reply:
Metadata api pulls down the &apos;, so I put my faith in that. It has been translated well in the past and is a very specific character.